styles.css

```diff
@@ -1,35 +0,0 @@
-body {
-    font-family: 'Poppins', sans-serif;
-    font-size: 1.8rem;
-    font-weight: 400;
-    line-height: 1.4;
-   
-}
-
-#intro{
-    padding-top: 20%;
-}
-#welcome-section{
-    background-color: #303841;
-}
-#projects{
-    background-color: #45567d;
-}
-#project-title{
-    text-decoration: underline;
-}
-.card-body{
-    background-color: #303841;
-}
-.card-text{
-    color: white;
-}
-#footer{
-    background-color: #303841;
-}
-.here:hover {
-	background-color: #45567d;
-}
-a{
-    text-decoration: none;
-}
```

Comment:
Please consider using Html&Css linters in your CSS to make it look perfect and clean which will save your coding time while reading and modifying the codes!